server/server.js

```diff
@@ -73,7 +105,13 @@ app.get(`${apiBaseUrl}/employer-form`,
   keycloak.protect(allowRoles('employer', 'health_authority', 'ministry_of_health')),
   asyncMiddleware(async (req, res) => {
     try {
-      const result = await dbClient.db[collections.EMPLOYER_FORMS].findDoc({});
+      const roles = getUserRoles(req);
+      const isHA = roles.includes('health_authority');
+      const isSuperUser = roles.includes('superuser');
+      const criteria = isHA && !isSuperUser ? getUserRegionsCriteria(req, 'healthAuthority') : {};
```

Comment:
I think this logic can be a little simpler, and also needs to include MOH

`const criteria = isSuperUser || isMOH ?  {} : getUserRegionsCriteria(req, 'healthAuthority');`

Basically:
- if superuser OR ministry_of_health, don't filter by region
- For all other cases (currently just HAs but will soon include employers), filter by region
:)